encrypt/uTests/symCrypt/AES256CBCTests.cpp

```diff
@@ -0,0 +1,41 @@
+#include <openssl/rand.h>
+
+#include "encrypt/symCrypt/AES256CBC.hpp"
+#include "gtest/gtest.h"
+
+using namespace encrypt;
+
+void generateAESKeyAndIV(std::vector<unsigned char> &key,
+                         std::vector<unsigned char> &iv) {
+  key.resize(32);  // AES 256-bit key size
+  if (RAND_bytes(key.data(), key.size()) != 1) {
+    throw std::runtime_error("Error generating AES key");
+  }
+
+  iv.resize(16);  // IV size for AES CBC mode
+  if (RAND_bytes(iv.data(), iv.size()) != 1) {
+    throw std::runtime_error("Error generating IV");
+  }
+}
+
+TEST(AES256CBCTests, simpleEncryptDecrypt) {
```

Comment:
This UT name is ok, though we could introduce a more standardized naming convention. Consider adopting a structure like "should" for expected behavior or "when...then" for specific conditions and outcomes.